docs/examples/audio/spectrogram.ipynb

```diff
@@ -86,7 +86,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "spectrogram_librosa_db = librosa.amplitude_to_db(spectrogram_librosa, ref=np.max)"
+    "spectrogram_librosa_db = librosa.power_to_db(spectrogram_librosa, ref=np.max)"
```

Comment:
Maybe we can plug AudioDecoder here?

Reply:
It can be done, but I see that as a change unrelated to the scope of this PR

Reply:
I'll create a follow up PR to adjust the example